src/components/categories/Categories.js

```diff
@@ -1,6 +1,34 @@
-export const Categories = () => { 
+import { useState, useEffect } from "react"
+import { useNavigate } from "react-router-dom"
+import { getCategories } from "../../managers/categories"
+import { Category } from "./Category"
+import "./Categories.css"
 
-    return( <>
-        <h1>Categories</h1>
-    </>)
+export const Categories = () => {
+
+    const [categories, setCategories] = useState([])
+    const navigate = useNavigate()
+
+    useEffect(
+        () => {
+            getCategories().then((categoryData) => {
```

Comment:
Since you're just passing along the response from the API call to another function without any other logic, consider rewriting this with more concise syntax

```js
getCategories().then(setCategories)
```

Reply:
Changes made.